src/Bicep.LangServer/Bicep.LangServer.csproj

```diff
@@ -18,4 +18,25 @@
     <EmbeddedResource Include="./Snippets/Templates/*.bicep" LogicalName="$([System.String]::new('Snippets/Templates/%(RecursiveDir)%(Filename)%(Extension)').Replace('\', '/'))" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Include="..\Bicep.Core\Configuration\bicepconfig.json">
```

Comment:
>bicepconfig [](http://example.com/codeflow?start=59&length=11)

Isn't this already packaged in Bicep.Core?

Reply:
Updated here:
https://github.com/Azure/bicep/commit/fe5c7a77d213a2beada95c437f88061c592ede4b

Thanks!